src/openassetio-core/include/openassetio/managerApi/ManagerInterface.hpp

```diff
@@ -1111,8 +1111,13 @@ class OPENASSETIO_CORE_EXPORT ManagerInterface {
    * that it is desired to publish the forthcoming data to. See the
    * notes in the API documentation for the specifics of this.
    *
-   * @param traitSet The @ref trait_set of the entities that are being
-   * published.
+   * @param entityTraitsDatas  @ref trait_set for each entity,
+   * determining the type of entity that the host will publish to this
+   * reference, complete with any property values that the host can
+   * provide ahead of time. This information should be used as a hint to
+   * help prepare for upcoming @ref resolve calls for
+   * @fqref{Context.Access.kWrite} "write access" and eventual @ref
+   * register to the associated entity in @p entityReferences.
```

Comment:
[consider] Might need a tweak to match what we update the `Manager` side to.

Can we also add notes on erroring when required properties are missing to the error callback section?

Reply:
Left as is for now, let me know if this needs to change with all the changes from daa8a1ac. Also added notes to the `errorCallback` docs as part of daa8a1ac.